contracts/core/FreeBets/FreeBetsHolder.sol

```diff
@@ -82,6 +89,23 @@ contract FreeBetsHolder is Initializable, ProxyOwned, ProxyPausable, ProxyReentr
         _removeUserFunding(_user, _collateral, _receiver);
     }
 
+    /// @notice Removes expired free bet funds from multiple users and transfers them to the owner
+    /// @dev This function can be called by anyone, but only works for funds that have passed their expiration time
+    /// @param _users Array of user addresses whose expired funds will be removed
+    /// @param _collateral The token address of the collateral to be removed
+    function removeExpiredUserFunding(address[] calldata _users, address _collateral) external notPaused nonReentrant {
+        for (uint256 index = 0; index < _users.length; index++) {
+            address _user = _users[index];
```

Comment:
skip this user if he has no free bet balance `balancePerUserAndCollateral[_user][_collateral] ==0`